app/assets/javascripts/autocomplete.coffee

```diff
@@ -11,7 +11,7 @@ update_textfield = (list_element) ->
   $('.js-autocomplete-resource-id').val($(list_element).data('res-id'))
 
 ready = ->
-  $('.js-autocomplete-textfield').on('focus input', ->
+  $('.js-autocomplete-textfield').on('change keyup', ->
```

Comment:
The Javascript was triggering a search after every single character was typed, which led to exhausting the GitHub Search API [rate limit](https://developer.github.com/v3/search/#rate-limit) of 30req/sec in some cases where a user was indecisive.